Goal: Remove documentation parts that do not apply to Debian

Fixes: ?

Status wrt upstream: Debian specific

Note: Some part should maybe go in the FHS patch

Index: samba-3.0.25c/docs/manpages/swat.8
===================================================================
--- samba-3.0.25c.orig/docs/manpages/swat.8	2007-08-26 13:07:03.425508276 +0200
+++ samba-3.0.25c/docs/manpages/swat.8	2007-08-26 13:08:58.919872259 +0200
@@ -111,60 +111,6 @@
 .RS 3n
 Print a summary of command line options.
 .RE
-.SH "INSTALLATION"
-.PP
-Swat is included as binary package with most distributions. The package manager in this case takes care of the installation and configuration. This section is only for those who have compiled swat from scratch.
-.PP
-After you compile SWAT you need to run
-make install
-to install the
-swat
-binary and the various help files and images. A default install would put these in:
-.TP 3n
-\(bu
-/usr/local/samba/sbin/swat
-.TP 3n
-\(bu
-/usr/local/samba/swat/images/*
-.TP 3n
-\(bu
-/usr/local/samba/swat/help/*
-.SS "Inetd Installation"
-.PP
-You need to edit your
-\fI/etc/inetd.conf \fR
-and
-\fI/etc/services\fR
-to enable SWAT to be launched via
-inetd.
-.PP
-In
-\fI/etc/services\fR
-you need to add a line like this:
-.PP
-swat 901/tcp
-.PP
-Note for NIS/YP and LDAP users - you may need to rebuild the NIS service maps rather than alter your local
-\fI /etc/services\fR
-file.
-.PP
-the choice of port number isn't really important except that it should be less than 1024 and not currently used (using a number above 1024 presents an obscure security hole depending on the implementation details of your
-inetd
-daemon).
-.PP
-In
-\fI/etc/inetd.conf\fR
-you should add a line like this:
-.PP
-swat stream tcp nowait.400 root /usr/local/samba/sbin/swat swat
-.PP
-Once you have edited
-\fI/etc/services\fR
-and
-\fI/etc/inetd.conf\fR
-you need to send a HUP signal to inetd. To do this use
-kill -1 PID
-where PID is the process ID of the inetd daemon.
 .SH "LAUNCHING"
 .PP
 To launch SWAT just run your favorite web browser and point it at "http://localhost:901/".
@@ -182,14 +128,11 @@
 This file must contain a mapping of service name (e.g., swat) to service port (e.g., 901) and protocol type (e.g., tcp).
 .RE
 .PP
-\fI/usr/local/samba/lib/smb.conf\fR
+\fI/etc/samba/smb.conf\fR
 .RS 3n
 This is the default location of the
 \fBsmb.conf\fR(5)
-server configuration file that swat edits. Other common places that systems install this file are
-\fI /usr/samba/lib/smb.conf\fR
-and
-\fI/etc/smb.conf \fR. This file describes all the services the server is to make available to clients.
+server configuration file that swat edits. This file describes all the services the server is to make available to clients.
 .RE
 .SH "WARNINGS"
 .PP
Index: samba-3.0.25c/docs/manpages/nmbd.8
===================================================================
--- samba-3.0.25c.orig/docs/manpages/nmbd.8	2007-08-26 13:07:03.425508276 +0200
+++ samba-3.0.25c/docs/manpages/nmbd.8	2007-08-26 13:08:58.919872259 +0200
@@ -113,11 +113,9 @@
 to answer any name queries. Adding a line to this file affects name NetBIOS resolution from this host
 \fBONLY\fR.
 .sp
-The default path to this file is compiled into Samba as part of the build process. Common defaults are
-\fI/usr/local/samba/lib/lmhosts\fR,
-\fI/usr/samba/lib/lmhosts\fR
-or
-\fI/etc/samba/lmhosts\fR. See the
+The default path to this file is
+\fI/etc/samba/lmhosts\fR.
+See the
 \fBlmhosts\fR(5)
 man page for details on the contents of this file.
 .RE
@@ -185,14 +183,11 @@
 inetd, this file must contain a mapping of service name (e.g., netbios-ssn) to service port (e.g., 139) and protocol type (e.g., tcp).
 .RE
 .PP
-\fI/usr/local/samba/lib/smb.conf\fR
+\fI/etc/samba/smb.conf\fR
 .RS 3n
 This is the default location of the
 \fBsmb.conf\fR(5)
-server configuration file. Other common places that systems install this file are
-\fI/usr/samba/lib/smb.conf\fR
-and
-\fI/etc/samba/smb.conf\fR.
+server configuration file.
 .sp
 When run as a WINS server (see the
 wins support parameter in the
@@ -234,10 +229,8 @@
 will accept SIGHUP, which will cause it to dump out its namelists into the file
 \fInamelist.debug \fR
 in the
-\fI/usr/local/samba/var/locks\fR
-directory (or the
-\fIvar/locks\fR
-directory configured under wherever Samba was configured to install itself). This will also cause
+\fI/var/run/samba\fR
+directory. This will also cause
 nmbd
 to dump out its server database in the
 \fIlog.nmb\fR
Index: samba-3.0.25c/docs/manpages/smbd.8
===================================================================
--- samba-3.0.25c.orig/docs/manpages/smbd.8	2007-08-26 13:07:03.425508276 +0200
+++ samba-3.0.25c/docs/manpages/smbd.8	2007-08-26 13:08:58.919872259 +0200
@@ -168,14 +168,11 @@
 inetd, this file must contain a mapping of service name (e.g., netbios-ssn) to service port (e.g., 139) and protocol type (e.g., tcp).
 .RE
 .PP
-\fI/usr/local/samba/lib/smb.conf\fR
+\fI/etc/samba/smb.conf\fR
 .RS 3n
 This is the default location of the
 \fBsmb.conf\fR(5)
-server configuration file. Other common places that systems install this file are
-\fI/usr/samba/lib/smb.conf\fR
-and
-\fI/etc/samba/smb.conf\fR.
+server configuration file.
 .sp
 This file describes all the services the server is to make available to clients. See
 \fBsmb.conf\fR(5)
Index: samba-3.0.25c/docs/manpages/lmhosts.5
===================================================================
--- samba-3.0.25c.orig/docs/manpages/lmhosts.5	2007-08-26 13:07:03.425508276 +0200
+++ samba-3.0.25c/docs/manpages/lmhosts.5	2007-08-26 13:08:58.919872259 +0200
@@ -77,10 +77,8 @@
 file.
 .SH "FILES"
 .PP
-lmhosts is loaded from the configuration directory. This is usually
-\fI/etc/samba\fR
-or
-\fI/usr/local/samba/lib\fR.
+lmhosts is loaded from the configuration directory. This is
+\fI/etc/samba\fR.
 .SH "VERSION"
 .PP
 This man page is correct for version 3.0 of the Samba suite.
Index: samba-3.0.25c/docs/manpages/ntlm_auth.1
===================================================================
--- samba-3.0.25c.orig/docs/manpages/ntlm_auth.1	2007-08-26 13:07:03.425508276 +0200
+++ samba-3.0.25c/docs/manpages/ntlm_auth.1	2007-08-26 13:08:58.919872259 +0200
@@ -43,7 +43,7 @@
 Some of these commands also require access to the directory
 \fIwinbindd_privileged\fR
 in
-\fI$LOCKDIR\fR. This should be done either by running this command as root or providing group access to the
+\fI/var/run/samba\fR. This should be done either by running this command as root or providing group access to the
 \fIwinbindd_privileged\fR
 directory. For security reasons, this directory should not be world-accessable.
 .SH "OPTIONS"
@@ -71,7 +71,7 @@
 Requires access to the directory
 \fIwinbindd_privileged\fR
 in
-\fI$LOCKDIR\fR. The protocol used is described here:
+\fI/var/run/samba\fR. The protocol used is described here:
 http://devel.squid-cache.org/ntlm/squid_helper_protocol.html. This protocol has been extended to allow the NTLMSSP Negotiate packet to be included as an argument to the
 YR
 command. (Thus avoiding loss of information in the protocol exchange).
@@ -94,7 +94,7 @@
 Requires access to the directory
 \fIwinbindd_privileged\fR
 in
-\fI$LOCKDIR\fR.
+\fI/var/run/samba\fR.
 .RE
 .PP
 gss-spnego-client
Index: samba-3.0.25c/docs/manpages/tdbbackup.8
===================================================================
--- samba-3.0.25c.orig/docs/manpages/tdbbackup.8	2007-08-26 13:07:03.425508276 +0200
+++ samba-3.0.25c/docs/manpages/tdbbackup.8	2007-08-26 13:08:58.919872259 +0200
@@ -70,17 +70,17 @@
 \(bu
 
 secrets.tdb
-- usual location is in the /usr/local/samba/private directory, or on some systems in /etc/samba.
+- in the /var/lib/samba directory.
 .TP 3n
 \(bu
 
 passdb.tdb
-- usual location is in the /usr/local/samba/private directory, or on some systems in /etc/samba.
+- in the /var/lib/samba directory.
 .TP 3n
 \(bu
 
 *.tdb
-located in the /usr/local/samba/var directory or on some systems in the /var/cache or /var/lib/samba directories.
+located in the /var/lib/samba and /var/run/samba directories.
 .SH "VERSION"
 .PP
 This man page is correct for version 3.0 of the Samba suite.
Index: samba-3.0.25c/docs/manpages/winbindd.8
===================================================================
--- samba-3.0.25c.orig/docs/manpages/winbindd.8	2007-08-26 13:07:03.425508276 +0200
+++ samba-3.0.25c/docs/manpages/winbindd.8	2007-08-26 13:08:58.919872259 +0200
@@ -440,16 +440,16 @@
 file are owned by root.
 .RE
 .PP
-$LOCKDIR/winbindd_privileged/pipe
+/var/run/samba/winbindd_privileged/pipe
 .RS 3n
 The UNIX pipe over which 'privileged' clients communicate with the
 winbindd
 program. For security reasons, access to some winbindd functions - like those needed by the
 ntlm_auth
-utility - is restricted. By default, only users in the 'root' group will get this access, however the administrator may change the group permissions on $LOCKDIR/winbindd_privileged to allow programs like 'squid' to use ntlm_auth. Note that the winbind client will only attempt to connect to the winbindd daemon if both the
-\fI$LOCKDIR/winbindd_privileged\fR
+utility - is restricted. By default, only users in the 'root' group will get this access, however the administrator may change the group permissions on /var/run/samba/winbindd_privileged to allow programs like 'squid' to use ntlm_auth. Note that the winbind client will only attempt to connect to the winbindd daemon if both the
+\fI/var/run/samba/winbindd_privileged\fR
 directory and
-\fI$LOCKDIR/winbindd_privileged/pipe\fR
+\fI/var/run/samba/winbindd_privileged/pipe\fR
 file are owned by root.
 .RE
 .PP
@@ -458,15 +458,12 @@
 Implementation of name service switch library.
 .RE
 .PP
-$LOCKDIR/winbindd_idmap.tdb
+/var/run/samba/winbindd_idmap.tdb
 .RS 3n
-Storage for the Windows NT rid to UNIX user/group id mapping. The lock directory is specified when Samba is initially compiled using the
-\fI--with-lockdir\fR
-option. This directory is by default
-\fI/usr/local/samba/var/locks \fR.
+Storage for the Windows NT rid to UNIX user/group id mapping.
 .RE
 .PP
-$LOCKDIR/winbindd_cache.tdb
+/var/run/samba/winbindd_cache.tdb
 .RS 3n
 Storage for cached user and group information.
 .RE
